sqlalchemy_redshift/dialect.py

```
@@ -1227,20 +1412,35 @@ def create_connect_args(self, *args, **kwargs):
         return cargs, default_args
 
     @classmethod
-    def dbapi(cls):
+    def import_dbapi(cls):
```

Ruff (0.14.0)

Missing return type annotation for classmethod `import_dbapi`

(ANN206)